Makefile

```
@@ -2,5 +2,9 @@ test:
 	composer update --prefer-dist --no-interaction ${COMPOSER_PARAMS}
 	composer test
 
+test-sf4:
+	composer require symfony/framework-bundle:"^4.0" --no-update
+	$(MAKE) test-lowest
```

Is there a reason to test on lowest once switched to `4.x`?
```suggestion
	$(MAKE) test
```

---

The above line only changes the composer.json file (due to `--no-uodate`) and after this change test lowest will update symfony to the lowest ^4.0 version, which fits other constrains

---

But, what would actually be the win of testing on the lowest 4.x version? That version isn't supported anymore. 

Testing the highest version of the 4.x branch would be great!